ironfish/src/primitives/__fixtures__/transaction.test.slow.ts.fixture

```diff
@@ -2,38 +2,38 @@
   "Accounts produces unique transaction hashes": [
```

Comment:
are you sure fixtures need to be updated for this change? we're just changing a static method on the Target class.